download_and_install/virtualbox.md

```diff
@@ -0,0 +1,114 @@
+OLA inside VirtualBox
+=====================
+
+These instructions explain how to install OLA inside a virtual machine.
+
+Downloads
+=========
+
+Download VirtualBox from [here](https://www.virtualbox.org/wiki/Downloads).
+
+Also download Debian from [here](https://www.debian.org/).
```

Comment:
What exactly? The latest iso? Which one?

Reply:
Linked to actual download page to clear this up.